sdks/java/core/src/test/java/org/apache/beam/sdk/util/HistogramDataTest.java

```diff
@@ -357,27 +357,74 @@ public void testStatistics_sumOfSquaredDeviations() {
 
   @Test
   public void testGetAndReset_resetSucceeds() {
-    HistogramData originalHistogram = HistogramData.linear(0, 10, 10);
+    // Records values from [20, 50) in three buckets that have width 10.
+    HistogramData originalHistogram = HistogramData.linear(20, 10, 3);
     originalHistogram.record(15.0, 25.0, 35.0, 45.0);
     originalHistogram.getAndReset();
 
-    HistogramData emptyHistogramData = HistogramData.linear(0, 10, 10);
+    HistogramData emptyHistogramData = HistogramData.linear(20, 10, 3);
```

Comment:
Is there a reason that needs to change these parameters? The original spacing should still pass the test, If not does it indicate some issue?

Reply:
The spacing for `originalHistogram` and `emptyHistogramData` need to be equal. If I used the original spacing for both histograms then it would still pass the test; however, I wanted to ensure these tests were incrementing the underflow/overflow buckets.